src/lib/service/hangar.ts

```diff
@@ -0,0 +1,17 @@
+export interface Result {
```

Comment:
This should be called something else rather than `Result` to be more descriptive

Reply:
Changed to `HangarProjectList`